public/js/monitor.js

```diff
@@ -160,6 +160,22 @@ function toggleHeaderStates(header, win) {
   }
 }
 
+function styleActiveLink(locationHref) {
+  let queryString = `.nav-link[href='${locationHref}']`;
+  const activeLink = document.querySelector(queryString);
+  if (activeLink) {
+    return activeLink.classList.add("active-link");
+  }
+
+  if (~locationHref.indexOf("/dashboard")) {
```

Comment:
Don't need to use bitwise OR (`~`) here. Let's just do `locationHref.indexOf("/dashboard") > -1` or `locationHref.indexOf("/dashboard") !== -1`

Reply:
I ain't gonna lie. I had to Bing exactly what the bitwise `~` operator was.
+1 to using `!locationHref.includes("/dashboard")` or what @groovecoder suggests above.
I also had to refresh my memory on the 2 logged in routes, which were:
- /user/preferences
- /user/dashboard

So my guess is that when the user is on /user/preferences, nothing will be highlighted, but that seems reasonable.

I can't really think of any other better solutions, unless it makes sense to check for the `/user/dashboard` route first, and overwrite it with the `/` route:

```js
function setActiveLink(locationHref) {
  if (locationHref === "/user/dashboard") {
    locationHref = "/";
  }
  const queryString = `.nav-link[href='${locationHref}']`;
  const activeLink = document.querySelector(queryString);
  if (activeLink) {
    return activeLink.classList.add("active-link");
  }
  ...
}
```

Reply:
There is also an ESLint [`no-bitwise`](https://eslint.org/docs/rules/no-bitwise) rule which disallows bitwise operators:

> **disallow bitwise operators** (`no-bitwise`)
> The use of bitwise operators in JavaScript is very rare and often `&` or `|` is simply a mistyped `&&` or `||`, which will lead to unexpected behavior.

Doesn't throw any existing errors in our code, not sure if we want to bother adding it to our ESLint config.